syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -243,12 +243,14 @@ def reconstruct(self):
 
         return sy.MultiPointerTensor(children=pointers)
 
-    def zero(self):
+    def zero(self, shape=None):
```

Comment:
Needed this in case I have a matrix multiplication - in that case the dimension for the zero shares might be different.